om4labs/om4common.py

```diff
@@ -262,6 +262,9 @@ def image_handler(figs, dictArgs, filename="./figure"):
         In-memory image buffers
     """
 
+    # Set default dpi
+    dpi = dictArgs["dpi"] if dpi in dictArgs.keys() else 100
```

Comment:
please consider

```python
dpi = dictArgs["dpi"] if dpi in dictArgs.keys() else default_dpi
```

and add `default_dpi=100` to the function kwargs

Reply:
Thanks @raphaeldussin  - see ea5b3db